CMakeLists.txt

```diff
@@ -626,8 +626,8 @@ if (NOT MINGW AND NOT MSVC AND NOT APPLE)
     configure_file(build/Linux+BSD/mscore.desktop.in mscore${MSCORE_INSTALL_SUFFIX}.desktop)
     install( FILES ${PROJECT_BINARY_DIR}/mscore${MSCORE_INSTALL_SUFFIX}.desktop DESTINATION share/applications)
     # Install appdata file (perform variable substitution first)
-    configure_file(build/Linux+BSD/mscore.appdata.xml.in mscore${MSCORE_INSTALL_SUFFIX}.appdata.xml)
-    install( FILES ${PROJECT_BINARY_DIR}/mscore${MSCORE_INSTALL_SUFFIX}.appdata.xml DESTINATION share/appdata)
+    configure_file(build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in org.musescore.MuseScore.appdata.xml)
+    install( FILES ${PROJECT_BINARY_DIR}/org.musescore.MuseScore.appdata.xml DESTINATION share/appdata)
```

Comment:
At least on Debian, the `*.appdata.xml` files must be installed into `share/metainfo` instead. Please double-check whether this is distro-specific or universal.

Reply:
I’ll check this – all the AppData in Flathub get installed into `share/appdata`, I _think_ `share/metainfo` is deprecated, but not sure.

Reply:
The CMake variable `${MSCORE_INSTALL_SUFFIX}` is there to allow developers to install more than one version at a time. It is empty by default, so it will not affect Flatpack releases if you put it back in.

Reply:
Yep, my fault, `share/metainfo` is the new directory, will fix.